### PR TITLE
Latest updates

### PR DESCRIPTION
**Pull request by ramil350**

### CODE REVIEW

File: lib/tasks/csp_reports_tasks.rake [hunk 1]
```diff
@@ -1,4 +1,24 @@
-# desc "Explaining what the task does"
-# task :csp_reports do
-#   # Task goes here
-# end
+namespace :csp_reports do
```
Comment (codelion) [inline lib/tasks/csp_reports_tasks.rake:1]: What does this rake task do? Is it only to populate sample data?

**Comment by codelion** [comment]
GTM except for the note I left.

File: lib/tasks/csp_reports_tasks.rake (hunk 1 above)
Comment (ramil350) [inline lib/tasks/csp_reports_tasks.rake:1]: @codelion yes, it is.

Comment (codelion) [inline lib/tasks/csp_reports_tasks.rake:1]: Thanks. merged!

**Pull request merged**